src/render.js

```diff
@@ -31,15 +29,19 @@ const resetGlobals = function() {
 
 resetGlobals();
 
-const runInContext = function(context, fn) {
-    return vm.runInContext("(" + fn.toString() + ")()", context);
-};
-
```

Comment:
No more duplication of this helper. Instead, we've attached a `run` method to the `context` that we create.